### ISSUE TITLE
How to edit faces on real face images

### ISSUE THREAD

**Issue by caiqiuurn** [genforce/interfacegan#61]
Thank you for providing your code. I wanna know whether your code can edit real face images.

**Comment by ShenYujun** [comment]
Yes, our codebase works well together with GAN inversion approaches, such as [mGANprior](https://genforce.github.io/mganprior/), [Image2StyleGAN](https://openaccess.thecvf.com/content_ICCV_2019/papers/Abdal_Image2StyleGAN_How_to_Embed_Images_Into_the_StyleGAN_Latent_Space_ICCV_2019_paper.pdf), [LIA](https://github.com/genforce/lia), [IDInvert](https://github.com/genforce/idinvert), etc.

**Comment by iPsych** [comment]
@ShenYujun Hi, Do you have any example or application of using interfacegan with mGANprior or other inversion approach?

**Comment by ShenYujun** [comment]
The boundaries provided in [mGANprior repo](https://github.com/genforce/mganprior) and [IDInvert repo](https://github.com/genforce/idinvert) are both obtained by InterFaceGAN.
